www/source/blog/tag.html.erb

```diff
@@ -0,0 +1,25 @@
+---
+pageable: true
+per_page: 12
+---
+<h1>Articles tagged '<%= tagname %>'</h1>
```

Comment:
Layout looks a little off, here -- I think we want this inside a `container blog` as well, like the index page, or to give it a similar container specific to tags.

Also, should we make these new views `.slim`s also? Seems like we should go with one or the other. (I don't have a preference on this, other than consistency.)